src/main/resources/org/clulab/asist/grammars/actions.yml

```diff
@@ -173,18 +158,38 @@ rules:
   - name: obstructing
     priority: ${ rulepriority }
     label: Block
+    example: "Some rubble is blocking this door"
     action: removeResearcher
     pattern: |
       trigger = [lemma=/(?i)^(${ block_triggers })/ & tag=/^V/]
-      source: Concept? = >/${agents}/ | <acl [incoming=nsubj]
+      source: Concept? = >/${agents}/
```

Comment:
The others you switched to `Obstacle` is this left generic as a backoff?

Reply:
yes